### PR TITLE
Improve RFC english syntax

### PR DESCRIPTION
**Pull request by woecifaun**
Hi,

I just wanted to help, contributing to your efforts.

I just packaged the changes in a single commit but I understand it could be harder to merge only interesting ones. I can definitely create a new PR with only wanted changes.

### CODE REVIEW

File: RFC.doku [hunk 1]
```diff
@@ -16,26 +18,30 @@ A deprecated feature may throw an exception before major release if the security
...
-The deprecated message for mysql is very clear: Deprecated: mysql_connect(): The mysql extension is deprecated and will be removed in the future: use mysqli or PDO instead. 
-But for ereg it's a little less obvious: Deprecated: Function ereg() is deprecated. If the standard deprecated message always include the replacement feature, 
-it will be easier to move forward to new versions of PHP.
+If the standard deprecated message always includes the replacing feature, it will be easier to move forward to new versions of PHP.
 
 
 ====== Exemple of normalized deprecated error ======
 
 
-"Deprecated: \[Feature name] : The feature \[feature name] is deprecated and will be removed in \[PHP Version]. Use \[replacement feature] instead."
```
Comment (woecifaun) [inline RFC.doku:32]: There's no space before colon in english (valid for other correction in the document).

File: RFC.doku [hunk 2]
```diff
@@ -16,26 +18,30 @@ A deprecated feature may throw an exception before major release if the security
...
+"Deprecated: \[Feature name]: The feature \[feature name] is deprecated and will be removed in \[PHP Version]. Use \[replacing feature] instead."
 
 
 ===== What we win =====
 
-* Removal of deprecated features are always in the roadmap for the next major version
-* No time loss debating whether the deprecated features should be removed now or postponed sine die.
+* Removals of deprecated features are always in the roadmap for the next major version
+* No time wasted debating whether the deprecated features should be removed now or postponed sine die.
```
Comment (woecifaun) [inline RFC.doku:44]: Should normally be `No time of loss`, but I didn't find that smooth.

File: RFC.doku [hunk 3]
```diff
@@ -16,26 +18,30 @@ A deprecated feature may throw an exception before major release if the security
 
 ====== Normalization of the deprecation message ======
 
-The deprecation message must include the version of PHP in which the feature will be removed. 
+The deprecation message must include the version of PHP in which the feature will be removed.
+
+===== Defining replacements =====
+
+Active php users — taking care of php evolutions — know which features have to be used instead of the old ones: mysqli vs mysql, preg vs ereg, etc.
```
Comment (woecifaun) [inline RFC.doku:25]: I replaced `in replacement for` which is quite heavy by `instead of`, but it doesn't have the same strengh.

**Comment by gorghoa** [comment]
Thanks! (with no space between ! and s ;-) )

**Pull request merged**